.github/workflows/linux_builds.yml

```
@@ -48,6 +48,16 @@ jobs:
             build-mono: false
             artifact: true
 
+          - name: Minimal Template (target=release, tools=no, everything disabled)
+            cache-name: linux-template-minimal
+            target: release
+            tools: false
+            tests: false
+            sconsflags: >
+              disable_3d=yes disable_advanced_gui=yes deprecated=no minizip=no xaudio2=no
+              modules_enabled_by_default=no
+            artifact: true
```

This is the configuration I used which worked well and matches the one above
```suggestion
          - name: Minimal template (target=release, tools=no, everything disabled)
            cache-name: linux-template-minimal
            target: release
            tools: false
            sconsflags: modules_enabled_by_default=no disable_3d=yes disable_advanced_gui=yes deprecated=no minizip=no debug_symbols=no
            build-mono: false
            artifact: true
```